[plugin-video] feat: settings panel to allow download toggle (#2229)
wix/ricos: website/docs/plugins_api/VideoPlugin.mdx

## Video Config API

### `Viewer disableDownload (deprecated!)`

> #### This option is no longer in available, if used the same behavior will remain. disableDownload should be set in the mediaSettings

Reviewer: ```suggestion
#### disableDownload should be set in the mediaSettings of the editor.
```
